packages/back-end/src/app.ts

```diff
@@ -150,6 +152,26 @@ app.get("/favicon.ico", (req, res) => {
   res.status(404).send("");
 });
 
+let robotsTxt = "";
+app.get("/robots.txt", (_req, res) => {
+  if (!robotsTxt) {
+    const file =
+      process.env.ROBOTS_TXT_PATH || path.join(__dirname, "..", "robots.txt");
+    if (existsSync(file)) {
+      robotsTxt = readFileSync(file).toString();
+    } else {
+      res.status(404).json({
+        message: "Not found",
+      });
+      return;
+    }
+  }
+
+  res.setHeader("Cache-Control", "max-age=3600");
+  res.setHeader("Content-Type", "text/yaml");
```

Comment:
@Kevin-Chant perhaps this should be `text/plain`?

Reply:
Good catch. That's what I get for copying existing code wholesale :sweat_smile: